ZeldaFullEditor/Data/Palettes.cs

```diff
@@ -413,106 +427,149 @@ private static Color GetColorShade(Color col, byte shade)
             return Color.FromArgb(r * 8, g * 8, b * 8);
         }
 
-		private static string SavePalettesToAsm(byte[] romData)
+        // I'm leaving this here and not moving it to constants because this is FG specific and should not be used by anyone else.
```

Comment:
rename the constants to say they are fairy garden specific with a prefix or something just to be explicit, maybe even a todo to make the addresses arguments to the function and provide a FairyGuardReloadOwPalettes function for yourself with these constants

Reply:
Done